src/tooltip.js

```diff
@@ -72,7 +72,7 @@ nv.models.tooltip = function() {
                 .attr("colspan", 3)
                 .append("strong")
                 .classed("x-value", true)
-                .html(headerFormatter(d.value));
+                .html(headerFormatter(d.value,d.index));
```

Comment:
I think I'd rather just pass d itself here... that way we don't have to keep adding more and more to the function call every time someone needs another thing off d.